### PR TITLE
[Tutorial/Basic] Translate documents in Korean (09,10,11,12,13) #140

### LINKED ISSUE #140: [Tutorial/Basic] Create Korean documents, 09 - 13
Create tutorial documents in Korean

- 09 adding-icons-kr.md
- 10 local-states-kr.md
- 11 add-items-kr.md
- 12 delete-items-kr.md
- 13 complete-items-kr.md

### PR DESCRIPTION
**Pull request by sonypark**
resolve #140

## Changes
- Create tutorial documents in Korean
- 09 adding-icons-kr.md
- 10 local-states-kr.md
- 11 add-items-kr.md
- 12 delete-items-kr.md
- 13 complete-items-kr.md

#### (Automated Tests)
- N/A

#### (Manual Tests)
- Test in local with jekyll

### Notes
- Request review on this PR to @Haerang94

### References
- N/A

### CODE REVIEW

File: docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+```
+todos: {id: Number, textValue: string, checked: boolean }
+```
+
+- `id`: 각 목록의 고유 아이디
+- `textValue`: 목록 내용
+- `checked`: 완료 여부. (`true`이면 완료 `false`이면 미완료)
+
+본 튜토리얼에서는 상태 관리를 위해서 훅(hook)을 사용하겠습니다.
```
Comment (JeffGuKang) [inline docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md:21]: I think linking official link could be help someone who want to find more about `hook`.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]

**Comment by JeffGuKang** [comment]
This is a very good quality tutorial. 💯

File: docs/basic-tutorial/basic-features(todolist)/09-adding-icons/adding-icons-kr.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+`react-native-vector-icons`는 다양한 아이콘을 제공합니다. 첨부한 링크에 들어가서 본인 취향에 따라 다양한 아이콘을 선택해 사용하시면 됩니다. [https://oblador.github.io/react-native-vector-icons/](https://oblador.github.io/react-native-vector-icons/)
+
+TodoListItem 컴포넌트에서 `react-native-vector-icons`를 임포트 하겠습니다.
+
+```js
+import Icon from 'react-native-vector-icons/AntDesign';
+```
+
+`StyleSheet`을 속성 안에 완료한 목록을 표시해주는 스타일을 추가해주겠습니다.
```
Comment (qkreltms) [inline docs/basic-tutorial/basic-features(todolist)/09-adding-icons/adding-icons-kr.md]: Changing "추가해주겠습니다." to "추가하겠습니다." would be better for me.

File: docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md (hunk 1 above)
Comment (sonypark) [reply docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md:21]: Yes I will add links on the post.

File: docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    </ScrollView>
+  );
+};
+```
+
+`map()` 함수를 이용해서 `todos`에 담긴 아이템을 하나씩 TodoListItem 컴포넌트로 전달합니다. 자바스크립트의 디스트럭처링(destructuring) 문법을 이용하면 TodoListItem 컴포넌트에서 아이템 객체에 담긴 값들을 바로 받을 수 있습니다.
+
+각각의 아이템에는 `textValue`, `id`, `checked`라는 **key**와 그에 해당하는 **value**가 담겨 있습니다.
+
+In TodoListItem component, we can get the values from props. Now let's put the textValue on Text area.
```
Comment (qkreltms) [inline docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md]: It seems English comment is added accidentally. Please check it.

File: docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+입력창에 텍스트를 입력하여 ADD 버튼을 눌러 추가하면 리스트에 아이템이 추가됩니다.
+
+![](../images/add_items.png "add_items.png")
+
+그런데 시뮬레이터 하단에 경고창이 뜨는 걸 볼 수 있습니다. 경고 문구는 다음과 같습니다. 
+
+`Each child in a list should have a unique 'key' prop.`
+즉, 리스트에 있는 아이템 객체마다 고유한 key값이 필요하다는 말 입니다.
+
+앞서 새로운 아이템 객체를 생성할 때 랜덤으로 id 값을 만들어준 것을 기억하시나요? id 값을 만들어 준 이유가 바로 여기 있습니다. 리스트에 아이템을 출력할 때에는 각 아이템의 고유 번호를 key 값으로 주어야기 때문입니다. (저희는 간단하게 `Math.random()`으로 id를 만들었지만, 이는 중복될 가능성이 있습니다. 따라서 실제 서비스를 만들 때에는 `uuid`와 같은 패키지 등을 이용하여 중복되지 않는 id를 생성해야합니다.)
```
Comment (qkreltms) [inline docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md]: External link for `UUID` package would be better for me.

**Review by qkreltms — CHANGES_REQUESTED** [review]

File: docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md (hunk 4 above)
Comment (sonypark) [reply docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md]: Very good suggestion, I added `uuid` package link to the sentence. Thank you!

File: docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md (hunk 3 above)
Comment (sonypark) [reply docs/basic-tutorial/basic-features(todolist)/11-add-items/add-items-kr.md]: Oh My bad, thank you so much!

File: docs/basic-tutorial/basic-features(todolist)/09-adding-icons/adding-icons-kr.md (hunk 2 above)
Comment (sonypark) [reply docs/basic-tutorial/basic-features(todolist)/09-adding-icons/adding-icons-kr.md]: I edited the sentence for better flow as you mentioned. Thank you

File: docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md (hunk 1 above)
Comment (sonypark) [reply docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md:21]: I added official link to the sentence.

**Review by JeffGuKang — APPROVED** [review]

**Pull request merged**